packages/@react-aria/checkbox/src/useCheckboxGroup.ts

```diff
@@ -33,7 +37,7 @@ export interface CheckboxGroupAria {
 export function useCheckboxGroup(props: AriaCheckboxGroupProps, state: CheckboxGroupState): CheckboxGroupAria {
```

Comment:
I don't think `useCheckboxGroup` actually accepts `validationState`, we've just been getting lucky with passing all props from `CheckboxGroup` it seems. Probably should update the `AriaCheckboxGroupProps` to support the stuff that goes into `AriaFieldProps` except for things like `labelElementType` (which we hard code).